Source/Microsoft.Teams.Apps.CompanyCommunicator/Authentication/AuthenticationServiceCollectionExtensions.cs

```diff
@@ -135,5 +136,39 @@ private static void RegisterAuthorizationPolicy(IServiceCollection services)
 
             services.AddSingleton<IAuthorizationHandler, MustBeValidUpnHandler>();
         }
+
+        private static bool AudienceValidator(
+            IEnumerable<string> audiences,
```

Comment:
Change name to tokenAudiences

(so it is really clear these are coming from the token)